python/setup.py

```diff
@@ -166,7 +166,19 @@ def get_package_data_files():
     ],
     extras_require={
         "test": ["pillow<7", "matplotlib"],
-        "extra_feature": ["tornado", "psutil", "xgboost>=1.1.0", "mypy", "orderedset"],
+        "extra_feature": [
+            "tornado",
+            "psutil",
+            "xgboost>=1.1.0",
+            "mypy",
+            "orderedset",
+            "tensorflow==2.1.0",
+            "tflite==2.1.0",
```

Comment:
ideally froma users' PoV, we don't want to restrict the dependency to a certain version. `>=` is better. Although we understand things are a moving target. 

We should express the current CI setups in a separate requirement setting.

Reply:
You mean having a separate `extras_require` label, such as `tvmc`, for example?

```
extras_require={
        "test": ["pillow<7", "matplotlib"],
        "extra_feature": [
            "tornado",
            "psutil",
            "xgboost>=1.1.0",
            "mypy",
            "orderedset",
        ],
        "tvmc": [
            "tensorflow>=2.1.0",
            "tflite>=2.1.0",
            "onnx>=1.7.0",
            "onnxruntime>=1.0.0",
            "torch>=1.4.0",
            "torchvision>=0.5.0",
        ],
...
```

Reply:
that looks good to me

Reply:
Done